libcloud/storage/drivers/local.py

```diff
@@ -276,7 +279,7 @@ def iterate_container_objects(self, container, prefix=None,
         """
         prefix = self._normalize_prefix_argument(prefix, ex_prefix)
 
-        objects = self._get_objects(container)
+        objects = self._get_objects(container, prefix)
         return self._filter_listed_container_objects(objects, prefix)
```

Comment:
If we now do the filtering natively in the driver I don't we need the fallback call to `_filter_listed_container_objects` anymore.

Reply:
Aight, removed the call to `_filter_listed_container_objects`